src/pages/ServerRoleSummary/UpdateNetworks.js

```diff
@@ -187,8 +188,9 @@ class UpdateNetworks extends Component {
     });
   }
 
-  getNetworkGroups = () => {
-    return this.props.model.getIn(['inputModel','network-groups']).map(e => e.get('name'))
+  getNetworkGroups = (props) => {
+    const model = props ? props.model : this.props.model;
```

Comment:
getNetworkGroups is only called in one place, where props is being supplied. I don't think we should be falling back to looking at this.props.model.